virtio-devices/src/lib.rs

```diff
@@ -39,6 +40,7 @@ pub mod mem;
 pub mod net;
 pub mod net_util;
 mod pmem;
+pub mod rate_limiter;
```

Comment:
Does this need to be pub mod?

Reply:
Good catch. No need to be public as 'rate_limiter' is only going to be used inside the 'virtio-device' crate.